tests/serviceBuilder.test.js

```diff
@@ -35,6 +35,68 @@ tap.test('serviceBuilder', test => {
     nock.enableNetConnect()
   })
 
+  test.test('getHeader', innerTest => {
+    const HEADER_MIA_KEY = 'miaheader'
+    const HEADER_MIA = { [HEADER_MIA_KEY]: 'foo' }
+
+    innerTest.test('injects Mia Header if isMiaHeaderInjected option is missing', async assert => {
+      const myServiceNameScope = nock('http://my-service-name')
```

Comment:
you can specify the headers of the request. See https://github.com/nock/nock#specifying-request-headers